src/test/java/jenkins/plugins/logstash/persistence/AbstractLogstashIndexerDaoTest.java

```diff
@@ -79,6 +79,8 @@ public void buildPayloadSuccessTwoLines() throws Exception {
 
   private AbstractLogstashIndexerDao getInstance() {
     return new AbstractLogstashIndexerDao() {
+      private static final long serialVersionUID = 1L;
```

Comment:
minor: it should not be added here https://github.com/rzwitserloot/lombok/wiki/WHY-NOT:-serialVersionUID 
if findbugs forces this can we add a suppression?

Reply:
remove it and added suppression. Haven't checked already existing places though